libs/langchain/langchain/llms/bedrock.py

```diff
@@ -404,6 +494,65 @@ def _call(
 
         return self._prepare_input_and_invoke(prompt=prompt, stop=stop, **kwargs)
 
+    async def _astream(
+        self,
+        prompt: str,
+        stop: Optional[List[str]] = None,
+        run_manager: Optional[CallbackManagerForLLMRun] = None,
```

Comment:
This should be "run_manager: Optional[AsyncCallbackManagerForLLMRun] = None," to be compatible with the method being overwritten in "BaseLLM".
https://github.com/DLOVRIC2/bedrock-async-support/blob/4fb265c501ab477bd143ed0e6710e2c77341d8d4/libs/langchain/langchain/llms/base.py#L484